datadog/dogstatsd/base.py

```diff
@@ -601,6 +600,74 @@ def disable_telemetry(self):
     def enable_telemetry(self):
         self._telemetry = True
 
+    def _parse_env_connection_overrides(self, host, port, socket_path):
+        dogstatsd_url = os.environ.get("DD_DOGSTATSD_URL")
+
+        if (
+            host == DEFAULT_HOST
+            and port == DEFAULT_PORT
+            and socket_path is None
+            and dogstatsd_url is not None
+        ):
+            parsed = urlparse(dogstatsd_url)
+            # If all values are defaults, prefer DD_DOGSTATSD_URL if present.
+            if parsed.scheme == "unix":
+                log.debug(
+                    "Found a DD_DOGSTATSD_URL matching the uds syntax, "
+                    "setting socket path %s.", dogstatsd_url
+                )
+                return host, port, dogstatsd_url
+
+            elif dogstatsd_url.startswith(WINDOWS_NAMEDPIPE_SCHEME):
+                log.debug(
+                    "DD_DOGSTATSD_URL is configured to utilize a windows named pipe, "
+                    "which is not currently supported by datadogpy. Falling back to "
+                    "alternate connection identifiers."
+                )
```

Comment:
In the spirit of our [documentation guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense), let the generic "unsupported scheme" message handle this case.

And AIUI this was the only place where `WINDOWS_NAMEDPIPE_SCHEME` was used, so that can be removed too.

```suggestion
```